### DISCUSSION TITLE
ERRO: cannot assign requested address. Is the version of this problem too low?

### DISCUSSION THREAD

**Discussion by yingbo-wu** [fluxcd/flux2#2511] [Q&A]
version: 0.15.1
This error occurs every 2-3 days.
Is this a version problem?
Does the new version have this problem?
Git Repository: Git + Openssh (Git Server)
Helm Repository: Harbor
Image Registry: Harbor
K8s Version: 1.22.6
OS: Debian 10

The Source Controller Pod can only be deleted each time this problem occurs!

**Comment by kingdonb** [comment]
Is your cluster being restarted coinciding with the incidence of this issue?

There have been major changes between 0.18 and 0.27, I recommend upgrading, but there is not enough information to say what is causing the error you mentioned.

It sounds like an error you get when pods are started before the cluster nodes are ready. I get a similar error (not in Flux) with some applications that I use, where they can only function after the pod is deleted, because somehow they come up when the cluster network is not ready, and that information (network interfaces listing) is apparently sticky inside of the pod for its entire lifetime.

We will need more information from you in order to assist with this. But I do not believe this is a Flux issue.

**Reply by kingdonb** [reply]
Please let us know more information about your cluster so we can follow up.
